app/views/outcomes-report/PageOutcomesReport.vue

```diff
@@ -200,6 +203,15 @@ export default {
     // TODO: date range
     async fetchOutcomesReportStats ({ kind, orgIdOrSlug, includeSubOrgs, country, startDate, endDate }) {
       let stats
+      if (['classroom', 'teacher'].includes(kind) && orgIdOrSlug) {
```

Comment:
does district having clan ? i don't see auto clan for district. the max one is school auto clan. I'm not sure what about it, since we do not have school reports (only having suborg as school) . so I only add classroom and teacher first